src/apps/Permissions/PermissionsView.js

```diff
@@ -0,0 +1,141 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonIcon,
+  ContextMenu,
+  ContextMenuItem,
+  DataView,
+  IconTrash,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import { usePermissions } from '../../contexts/PermissionsContext'
+import LocalIdentityBadge from '../../components/IdentityBadge/LocalIdentityBadge'
+
+function PermissionsView({ permissions, onOpenEntity, heading, showApps }) {
+  const { layoutName } = useLayout()
+
+  const fields = [
+    'Action',
+    'On app',
+    { label: 'Assigned to entity', childStart: true },
+    'Managed by',
+  ]
+
+  if (!showApps) {
+    fields.splice(1, 1)
+  }
+
+  return (
+    <DataView
+      heading={heading}
+      mode={
```

Comment:
`DataView` does this automatically no?

Reply:
Yes, but not in the same way: https://github.com/aragon/aragon-ui/blob/newstyle/src/components/DataView/DataView.js#L206